src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/SubmissionApiDelegateImpl.java

```diff
@@ -430,4 +434,35 @@ private ResponseEntity storeDocuments(SubmissionKey submissionKey, List<Multipar
         return ResponseEntity.ok(new UploadSubmissionDocumentsResponse().submissionId(submissionKey.getSubmissionId()).received(new BigDecimal(files.size())));
     }
 
+    @Override
+    @RolesAllowed("efiling-user")
+    public ResponseEntity<Void> postRushProcessing(UUID xTransactionId, UUID submissionId, Rush rush) {
+
+        Optional<String> universalId = SecurityUtils.getUniversalIdFromContext();
+
+        if(!universalId.isPresent()) {
+
+            logger.error(UNIVERSAL_ID_IS_REQUIRED);
+            throw new InvalidUniversalException(INVALID_UNIVERSAL_ID);
+
+        }
+
+        SubmissionKey submissionKey = new SubmissionKey(universalId.get(), xTransactionId, submissionId);
+
+        logger.info("attempting to add rush processing to efiling package for transaction [{}]", xTransactionId);
```

Comment:
```suggestion
        logger.info("Attempting to add rush processing to efiling package for transaction [{}]", xTransactionId);
```

Reply:
is the transaction ID too much for prod logging?

Reply:
No and we are logging it everywhere. It doesn't get stored in the database

Reply:
Also that comment is consistent with others